### PR TITLE
python3-jsonschema: update to 4.14.0.

### PR DESCRIPTION
**Pull request by jcgruenhage**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/python3-jsonschema/template [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Template file for 'python3-jsonschema'
 pkgname=python3-jsonschema
-version=4.9.1
+version=4.12.1
 revision=1
 wrksrc="jsonschema-${version}"
 build_style=python3-pep517
-hostmakedepends="hatchling hatch-vcs"
+hostmakedepends="hatchling hatch-vcs hatch-fancy-pypi-readme"
```
Comment (paper42) [inline srcpkgs/python3-jsonschema/template:7]: If this is just generating a pypi readme (that we don't even install), can we just patch this out?

Comment (jcgruenhage) [reply srcpkgs/python3-jsonschema/template:7]: We could, but I don't see where that would be better. Maintaining this additional small package is probably less effort than regularly having to rebase a patch, and if others want to use this package it's now available, both for other void packages and for users directly.

Comment (paper42) [reply srcpkgs/python3-jsonschema/template:7]: I still think that maintaining a whole package with this specific usecase is not nice and we should just use a simple patch/vsed, but I will leave this to other maintainers.

Comment (classabbyamp) [reply srcpkgs/python3-jsonschema/template:7]: I say patch or vsed, the package doesn't provide much value to the distro

Comment (jcgruenhage) [reply srcpkgs/python3-jsonschema/template]: patched it out, because it's too big for vsed IMO. Also: 9 releases in 5 days :/

**Pull request merged**